(doc) Document updating manual packages (#2671)
chocolatey-community/chocolatey-packages: COMMITTERS.md

Before providing a fixed package version, you need to determine the existing version specified in the repository, what type of version it uses, and whether the package contains additional handling for creating fixed versions.

### Updating a Manual Package

> A manual package is one that is not using the Chocolatey-AU automation. The specific package files should be updated normally (i.e. the `.nuspec` file and the Chocolatey package scripts). Once those are updated, and the pull request merged, the package is pushed to the Chocolatey Community Repository by makeing an empty commit, with the mesage int he format `[PUSH <PACKAGENAME>]` where `<PACKAGENAME>` is the name of the package you want to push.

Reviewer: ```suggestion
A manual package is one that is not using the Chocolatey-AU automation. The specific package files should be updated normally (i.e. the `.nuspec` file and the Chocolatey package scripts). Once those are updated, and the pull request merged, the package is pushed to the Chocolatey Community Repository by makeing an empty commit, with the mesage in the format `[PUSH <PACKAGENAME>]` where `<PACKAGENAME>` is the name of the package you want to push.
```

Reviewer 2: In addition to what @corbob has suggested, can we change `mesage` to be `message`?

Reviewer: I've added @gep13's suggestion into the original suggestion.

Author: Apologies for my abysmal spelling. Fixed.

> 

Reviewer: I'm not sure if it makes sense in this PR specifically, but since we discussed it today I thought I would drop it here as a suggestion. If we want to do some more wordsmithing of it, I'm more than happy to open my own PR for this specific information.

```suggestion

If the manual package is an embedded, or semi-embedded package, the mechanism for updating the files in the package is an `update.ps1` script beside the `.nuspec` file. This file will be run when the package is pushed, but files will not be committed back to the repository.

```
